database/mssql/scripts/versions/v_37_ddl.sql

```diff
@@ -0,0 +1,125 @@
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+SET NOCOUNT ON
+GO
+
+SET XACT_ABORT ON
+GO
+SET TRANSACTION ISOLATION LEVEL SERIALIZABLE
+GO
+BEGIN TRANSACTION
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+CREATE TABLE [permit].[ORBC_SPECIAL_AUTH](
+	[ID] [int] IDENTITY(1,1) NOT NULL,
+	[COMPANY_ID] [int] NOT NULL,
+	[LCV] [char](1) NOT NULL,
```

Comment:
Should there be a default value as the col is set as NOT NULL?

Reply:
Actually , it should be not null. Will change.

Reply:
done